src/storage/v3/edge_accessor.cpp

```diff
@@ -128,12 +129,12 @@ Result<PropertyValue> EdgeAccessor::GetProperty(PropertyId property, View view)
         break;
     }
   });
-  if (!exists) return Error::NONEXISTENT_OBJECT;
-  if (!for_deleted_ && deleted) return Error::DELETED_OBJECT;
+  if (!exists) return SHARD_ERROR(common::ErrorCode::NONEXISTENT_OBJECT);
+  if (!for_deleted_ && deleted) return SHARD_ERROR(common::ErrorCode::DELETED_OBJECT);
```

Comment:
What about including some of the prefix (`common::` or `ErrorCode::`) into the `SHARD_ERROR` macro or declaring it with `using ErrorCode = common::ErrorCode;` just to keep it short and thus easier to read?